app/controllers/employees/kudos_controller.rb

```diff
@@ -25,7 +25,8 @@ def create
         @kudo.save!
         @current_employee.number_of_available_kudos -= 1
         @current_employee.save!
-
+        @kudo.receiver.number_of_earned_points += 10
+        @kudo.receiver.save!
```

Comment:
What do you think about adding transactions to every place when you are saving multiple objects at once?

We do not want to have a situation when we remove available_kudo from current_employee and do not give earned_points to the receiver. And this can happen if something breaks between these two saves. Transaction will prevent this, and rollback all changes if something bad happen